### DISCUSSION TITLE
M1源码编译运行apisix报如下错误，请问有人碰到过吗，在错误日志中

### DISCUSSION THREAD

**Discussion by liuhuanyu** [apache/apisix#8443] [Q&A]
![image](https://user-images.githubusercontent.com/5518744/205082100-67698a0a-e0e5-465b-96ed-7d52fa1f6f6a.png)

**Comment by spacewander** [comment]
OpenResty 1.21.4.1 doesn't support M1

**Reply by tokers** [reply]
@liuhuanyu How did you install APISIX?

**Reply by liuhuanyu** [reply]
通过替换openresty 1,19.3解决了问题，是从源码安装的apisix

**Comment by liugang594** [comment]
可以看看这个帖子

https://github.com/apache/apisix/issues/7313
